### ISSUE TITLE
OMP: Error #15? Help?

### ISSUE THREAD

**Issue by pepsidude1** [alievk/avatarify#436]
Hi, im trying to download avatarify on windows, but when i do the, run_windows.bat i get this error:

OMP: Error #15: Initializing libiomp5md.dll, but found libiomp5md.dll already initialized.
OMP: Hint This means that multiple copies of the OpenMP runtime have been linked into the program. That is dangerous, since it can degrade performance or cause incorrect results. The best thing to do is to ensure that only a single OpenMP runtime is linked into the process, e.g. by avoiding static linking of the OpenMP runtime in any library. As an unsafe, unsupported, undocumented workaround you can set the environment variable KMP_DUPLICATE_LIB_OK=TRUE to allow the program to continue to execute, but that may cause crashes or silently produce incorrect results. For more information, please see http://www.intel.com/software/products/support/.

I tried doing the: import os 
os.environ["KMP_DUPLICATE_LIB_OK"]="TRUE"

But then i got the error:

'import' is not recognized as an internal or external command,
operable program or batch file.

and 

["KMP_DUPLICATE_LIB_OK"]="TRUE"
'os.environ["KMP_DUPLICATE_LIB_OK"]' is not recognized as an internal or external command,
operable program or batch file.

Can anyone help me?

**Comment by pepsidude1** [comment]
> The only install instructions that mention downloading nvidia driver is for running in Docker, which is for Linux. Make sure that you only follow steps 1-12 under the "Windows" section.
> 
> Did you run step 4 and 6 in the miniconda window that you opened in step 3? I've seen some being confused by that, and trying to run things in a regular command prompt.

Yes, i pasted it in miniconda i followed this video https://www.youtube.com/watch?v=lym9ANVb120

**Comment by Barchtic** [comment]
Getting the same issue here as well. I've gone through #411 solution. Currently on Windows 11 with a RTX 3090.
